src/main/flight/pid.h

```diff
@@ -82,6 +82,7 @@ typedef struct pidProfile_s {
 
 extern int16_t axisPID[XYZ_AXIS_COUNT];
 extern int32_t axisPID_P[3], axisPID_I[3], axisPID_D[3];
+bool antiWindupProtection;
```

Comment:
i'm not a fan of all these externs in pid.c. some kind of pid loop state structure for them would likely be nicer.